src/ansys/aedt/core/workflows/circuit/import_schematic.py

```diff
@@ -38,26 +39,27 @@
 
 # Extension batch arguments
 extension_arguments = {"asc_file": ""}
-extension_description = "Import schematic to Circuit."
+extension_description = "Import schematic to Circuit"
 
 
 def frontend():  # pragma: no cover
-
     import tkinter
     from tkinter import filedialog
     from tkinter import ttk
```

Comment:
```suggestion
    import tkinter as tk
    from tkinter import filedialog
    import tkinter.ttk as ttk
```
@Samuelopez-ansys @SMoraisAnsys for me this is better. What do you think?

Reply:
Let's do it in a different PR